tests/system/action/motion/test_create_sequence_number.py

```diff
@@ -0,0 +1,256 @@
+import threading
+from time import sleep
+
+from openslides_backend.shared.lock import (
+    monkeyPatch_DatastoreAdapter_write,
+    pytest_thread_local,
+)
+from tests.system.action.base import BaseActionTestCase
+
+
+class MotionCreateActionTestSequenceNumber(BaseActionTestCase):
+    def create_workflow(self, workflow_id: int = 12, meeting_id: int = 222) -> None:
+        state_id = workflow_id + 100
+        state_str = str(state_id)
+
+        self.create_model(
+            "motion_workflow/" + str(workflow_id),
+            {
+                "name": "name_workflow1",
+                "first_state_id": state_id,
+                "state_ids": [state_id],
+            },
+        )
+        self.create_model(
+            "motion_state/" + state_str,
+            {"name": "name_state" + state_str, "meeting_id": meeting_id},
+        )
+
+    def test_create_sequence_numbers(self) -> None:
+        self.create_model("meeting/222", {"name": "meeting222"})
+        self.create_workflow(workflow_id=12, meeting_id=222)
+
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "motion.create",
+                    "data": [
+                        {
+                            "title": "motion_title",
+                            "meeting_id": 222,
+                            "workflow_id": 12,
+                            "text": "test",
+                        }
+                    ],
+                }
+            ],
+        )
+        self.assert_status_code(response, 200)
+        model = self.get_model("motion/1")
+        self.assertEqual(model.get("sequence_number"), 1)
+
+        self.create_workflow(workflow_id=13, meeting_id=222)
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "motion.create",
+                    "data": [
+                        {
+                            "title": "motion_title2",
+                            "meeting_id": 222,
+                            "workflow_id": 13,
```

Comment:
why are you creating a different workflow here? It should not make a difference, if you use workflow 12.

Reply:
Done